From d5a3e8be1901fa07640452992c8a3ab36605f0fe Mon Sep 17 00:00:00 2001
From: William Wu <william.wu@rock-chips.com>
Date: Tue, 11 Dec 2018 11:38:23 +0800
Subject: [PATCH] usb: gadget: Kconfig: disable
 CONFIG_USB_CONFIGFS_F_UVC_ROCKCHIP

If we enable CONFIG_USB_CONFIGFS_F_UVC_ROCKCHIP, the userspace
must update the data length of v4l2_event to 4100 synchronously.
Because we don't modify the v4l2_event struct in the userspace
by default, so disable this config. You can enable this config
if you have modified the data length of v4l2_event to 4100 in
the userspace.

Change-Id: I68c354b2d8acba669b5c7bf1a425fdc34c05f1e5
Signed-off-by: William Wu <william.wu@rock-chips.com>
---
 drivers/usb/gadget/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/gadget/Kconfig b/drivers/usb/gadget/Kconfig
index 3bcd31a49772..2c2ef4277e91 100644
--- a/drivers/usb/gadget/Kconfig
+++ b/drivers/usb/gadget/Kconfig
@@ -490,7 +490,7 @@ config USB_CONFIGFS_F_UVC
 config USB_CONFIGFS_F_UVC_ROCKCHIP
 	bool "Rockchip USB Webcam function"
 	depends on USB_CONFIGFS_F_UVC && ARCH_ROCKCHIP
-	default y
+	default n
 	help
 	  The Rockchip USB Webcam function is used for Rockchip ISP IQ tool.
 	  Enable this option will increase the UVC_MAX_REQUEST_SIZE from
-- 
2.35.3

